dm/tests/_utils/run_downstream_cluster

```diff
@@ -24,63 +24,97 @@ start_pd() {
 max-replicas = 1
 EOF
 
-	pd-server --version
+	bin/pd-server --version
 	mkdir -p "$WORK_DIR/pd"
-	bin/pd-server \
+	nohup bin/pd-server \
 		--client-urls "http://$PD_ADDR" \
 		--peer-urls "http://$PD_PEER_ADDR" \
 		--log-file "$WORK_DIR/pd.log" \
 		--config "$WORK_DIR/pd.toml" \
-		--data-dir "$WORK_DIR/pd" &
-	# wait until PD is online...
+		--data-dir "$WORK_DIR/pd" >/dev/null 2>&1 &
+	sleep 5
 	i=0
-	while ! curl "http://$PD_ADDR/pd/api/v1/version"; do
+	while true; do
+		response=$(curl -s -o /dev/null -w "%{http_code}" "http://$PD_ADDR/pd/api/v1/version")
+		echo "curl response: $response"
+		if [ "$response" -eq 200 ]; then
+			echo 'Start PD success'
+			break
+		fi
 		i=$((i + 1))
 		if [ "$i" -gt 20 ]; then
 			echo 'Failed to start PD'
 			return 1
 		fi
+		echo 'Waiting for PD ready...'
 		sleep 3
 	done
 }
 
 start_tikv() {
 	echo "Starting TiKV..."
 	mkdir -p "$WORK_DIR/tikv"
-	bin/tikv-server \
+	bin/tikv-server --version
+	nohup bin/tikv-server \
 		--pd "$PD_ADDR" \
 		-A "$TIKV_ADDR" \
 		--status-addr "$TIKV_STATUS_ADDR" \
 		--log-file "$WORK_DIR/tikv.log" \
 		--log-level info \
-		-s "$WORK_DIR/tikv" &
-	while ! curl "http://$PD_ADDR/pd/api/v1/cluster/status" | tee /dev/stderr | grep '"is_initialized": true'; do
+		-s "$WORK_DIR/tikv" >/dev/null 2>&1 &
+	sleep 5
+	i=0
+	while true; do
+		response=$(curl -s "http://$PD_ADDR/pd/api/v1/cluster/status" || echo "")
+
+		if [ -z "$response" ]; then
+			echo "Failed to connect to PD server"
+		else
+			echo "PD response: $response"
+			if echo "$response" | grep -q '"is_initialized": true'; then
+				echo "TiKV cluster initialized successfully"
+				break
+			fi
+		fi
+
 		i=$((i + 1))
 		if [ "$i" -gt 20 ]; then
-			echo 'Failed to initialize TiKV cluster'
+			echo 'Failed to initialize TiKV cluster after 20 attempts'
+			echo "Last response: $response"
```

Comment:
last response is already printed in above code, don't need to print it again

Reply:
fixed by remove it.